src/io.c

```diff
@@ -407,9 +407,13 @@ dispatch_io_create_f(dispatch_io_type_t type, dispatch_fd_t fd,
 }
 
 #if defined(_WIN32)
-#define _is_separator(ch) ((ch) == '/' || (ch) == '\\')
+#define _is_separator(ch) ((ch) == L'/' || (ch) == L'\\')
+#define _dispatch_stat _wstat
```

Comment:
The only thing that I wonder about here is if we should be using `_wstat` or `_wstati64` to get 64-bit times and lengths.

Reply:
Not sure if I'm following🤔 I see no usage of time/size fields. But as a future-proof change it seems reasonable. That would be a simple drop-in replacement, as everything compiles without warnings and works same way. Should I push it?

Reply:
Yes please! The idea is to be prepared in case we start using that.

Reply:
Done